virl/files/mitaka.vinstall.py

```diff
@@ -1005,16 +993,12 @@ def determine_upgrade_type():
                  ospassword, '--os-user-domain-id=default', '--os-project-domain-id=default', '--os-auth-url=http://localhost:5000/{0}'.format(keystone_auth_version)]
         nmcall = ['nova-manage', '--os-tenant-name', 'admin', '--os-username', 'admin', '--os-password',
                  ospassword, '--os-auth-url=http://localhost:5000/{0}'.format(keystone_auth_version)]
-        subprocess.call(qcall + ['subnet-delete', 'flat'])
-        subprocess.call(qcall + ['subnet-delete', 'flat1'])
-        subprocess.call(qcall + ['subnet-delete', 'ext-net'])
-        q_delete_list = (subprocess.check_output( ['neutron --os-username admin --os-password {ospassword} --os-tenant-name admin --os-user-domain-id=default --os-project-domain-id=default --os-auth-url=http://localhost:5000/{authver} agent-list | grep -v "{hostname}" |grep -v "region" | grep -v "+-" | cut -d "|" -f2'.format(ospassword=ospassword,hostname=hostname,authver=keystone_auth_version)], shell=True)).split()
+        q_delete_list = (subprocess.check_output( ['neutron --os-username admin --os-password {ospassword} --os-tenant-name admin --os-user-domain-id=default --os-project-domain-id=default --os-auth-url=http://localhost:5000/{authver} agent-list | grep -v "+-" | sed 1d | grep -v "{hostname}" | grep -v "region" | cut -d "|" -f2'.format(ospassword=ospassword,hostname=hostname,authver=keystone_auth_version)], shell=True)).split()
```

Comment:
Original version did not strip header of neutron output, so q_delete_list contained header column. Commit fixed this issue only. Other issue is that this code greps hostname in whole line (not single column) what can lead to unwanted results. I could fix this too but i am not sure about other parts of this command (grep -v 'region') so i left it be.